backend/src/graphql/arg/TransactionLinkFilters.ts

```diff
@@ -3,11 +3,11 @@ import { ArgsType, Field } from 'type-graphql'
 @ArgsType()
 export default class TransactionLinkFilters {
   @Field(() => Boolean, { nullable: true, defaultValue: true })
-  filterByDeleted?: boolean
+  byDeleted?: boolean
 
   @Field(() => Boolean, { nullable: true, defaultValue: true })
-  filterByExpired?: boolean
+  byExpired?: boolean
 
   @Field(() => Boolean, { nullable: true, defaultValue: true })
-  filterByRedeemed?: boolean
+  byRedeemed?: boolean
```

Comment:
This seems to be three times the same:

- nullable
- `?` optional
- `| null`

Reply:
We can remove `?` and `| null`, but not the Field parameters.
Done